framework/src/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -452,6 +452,61 @@ protected String standardizeString(
     public boolean hasDependentAnnotations() {
         return !annoToElements.isEmpty();
     }
+    /**
+     * Standardizes an annotation. If the annotation is not an expression annotation, returns the
```

Comment:
Can you add an explanation of what `expression annotation` means, e.g. to the class doc? Isn't `dependent type annotation` the term you mean? What's the difference between the two?